mobile/src/main/java/mycroft/ai/MainActivity.java

```diff
@@ -452,8 +459,30 @@ protected void checkIfLaunchedFromWidget(Intent intent) {
         }
     }
 
+    private void loadVersionInfo() {
+        String versionName = "";
+        int versionCode = -1;
+        try {
+            PackageInfo packageInfo = getPackageManager().getPackageInfo(getPackageName(), 0);
+            versionName = packageInfo.versionName;
+            versionCode = packageInfo.versionCode;
+
+            SharedPreferences.Editor editor = sharedPref.edit();
+            editor.putInt("versionCode", versionCode);
+            editor.putString("versionName", versionName);
+            editor.commit();
+        } catch (PackageManager.NameNotFoundException e) {
+            e.printStackTrace();
+        }
+    }
+
+    private void loadLicenseInfo(){
+        SharedPreferences.Editor editor = sharedPref.edit();
+        editor.putString("license", getString(R.string.license_value));
```

Comment:
Why are we using java to store the license type in shared preferences? Shouldn't that already be set in the appropriate pref xml file?